packages/client/wallets/aa/src/services/logging/DatadogProvider.ts

```diff
@@ -0,0 +1,39 @@
+import { datadogLogs } from "@datadog/browser-logs";
+
+
```

Comment:
double space–I think prettier is not running

Reply:
I just copy and paste the file. I think this rule isn't set. Anyway, the suggestions make sense to me!